driver-core/src/main/com/mongodb/internal/binding/ClusterBinding.java

```diff
@@ -81,6 +88,11 @@ public SessionContext getSessionContext() {
         return new ReadConcernAwareNoOpSessionContext(readConcern);
     }
 
+    @Override
+    public ServerApi getServerApi() {
```

Comment:
Since this has to make it's way all the way down to CommandMessage, it seems best to add it to bindings rather than having to pass it down through every operation.